### PR TITLE
refactor(flag): remove old approach for flagged proposals

### PR DESCRIPTION
**Pull request by Todmy**
Remove backwards compatibility for fetching flagged proposals and flagged/verified spaces from Laser DB

### CODE REVIEW

**Review by wa0x6e — APPROVED** [review]
utAck

File: src/graphql/operations/proposals.ts [hunk 1]
```diff
@@ -61,21 +60,16 @@ export default async function (parent, args) {
...
+  if (where.space_verified) {
+    searchSql += ' AND spaces.verified = 1';
   }
 
-  // TODO: remove part `p.id IN (?)` when flagged proposals are moved from laser DB to snapshot-sequencer DB
-  if (where.flagged === true && flaggedProposals.length > 0) {
-    searchSql += ' AND (p.id IN (?) OR p.flagged = 1)';
-    params.push(flaggedProposals);
+  if (where.flagged === true) {
+    searchSql += ' AND p.flagged = 1)';
```
Comment (bonustrack) [inline src/graphql/operations/proposals.ts:68]: Was this tested?

File: src/graphql/operations/proposals.ts [hunk 2]
```diff
@@ -61,21 +60,16 @@ export default async function (parent, args) {
...
   }
 
-  // TODO: remove part `p.id NOT IN (?)` when flagged proposals are moved from laser DB to snapshot-sequencer DB
-  if (where.flagged === false && flaggedProposals.length > 0) {
-    searchSql += ' AND (p.id NOT IN (?) AND p.flagged = 0)';
-    params.push(flaggedProposals);
+  if (where.flagged === false) {
+    searchSql += ' AND p.flagged = 0)';
```
Comment (wa0x6e) [inline src/graphql/operations/proposals.ts:72]: Extra `)`

File: src/graphql/operations/proposals.ts (hunk 1 above)
Comment (wa0x6e) [inline src/graphql/operations/proposals.ts:68]: Extra `)`

**Review by wa0x6e — CHANGES_REQUESTED** [review]
tAck, works aside the 2 extra `)` in the sql query

**Review by Todmy — COMMENTED** [review]
fix(mysql): mistake in query

**Review by Todmy — COMMENTED** [review]
fix(mysql): mistake in query

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**